react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/views/ComposeViewWrapper.kt

```diff
@@ -35,8 +63,16 @@ abstract class ComposeViewWrapper<T : View> : FrameLayout {
     }
 
     override fun onAttachedToWindow() {
+        // CRITICAL: Set Compose owners on React Native modal container
+        // This fixes the issue where Compose views crash in React Native modals
+        // because it looks like modals create isolated view hierarchies without
+        // the required owners and it will crash with:
```

Comment:
This seems incomplete, but we can probably just leave it to the next line with the link to the github issue?